### ISSUE TITLE
Add rake as dependency

### ISSUE THREAD

**Issue by dotcomboom** [neocities/neocities-ruby#12]
On a i386 Debian distro (MX Linux non-PAE).
`ruby -v` returns `ruby 2.3.3p222 (2016-11-21) [i386-linux-gnu]`
```
~ $ sudo apt install ruby-full && sudo gem install neocities
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  libgmp-dev libgmpxx4ldbl ri ruby-dev ruby-tcltk ruby2.3-dev ruby2.3-doc ruby2.3-tcltk
Suggested packages:
  gmp-doc libgmp10-doc libmpfr-dev
The following NEW packages will be installed:
  libgmp-dev libgmpxx4ldbl ri ruby-dev ruby-full ruby-tcltk ruby2.3-dev ruby2.3-doc ruby2.3-tcltk
0 upgraded, 9 newly installed, 0 to remove and 0 not upgraded.
Need to get 5,915 kB of archives.
After this operation, 36.4 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://ftp.us.debian.org/debian stretch/main i386 libgmpxx4ldbl i386 2:6.1.2+dfsg-1 [22.6 kB]
Get:2 http://ftp.us.debian.org/debian stretch/main i386 libgmp-dev i386 2:6.1.2+dfsg-1 [645 kB]
Get:3 http://ftp.us.debian.org/debian stretch/main i386 ruby2.3-doc all 2.3.3-1+deb9u2 [3,512 kB]
Get:4 http://ftp.us.debian.org/debian stretch/main i386 ri all 1:2.3.3 [9,476 B]                                
Get:5 http://ftp.us.debian.org/debian stretch/main i386 ruby2.3-dev i386 2.3.3-1+deb9u2 [1,271 kB]              
Get:6 http://ftp.us.debian.org/debian stretch/main i386 ruby-dev i386 1:2.3.3 [9,574 B]                         
Get:7 http://ftp.us.debian.org/debian stretch/main i386 ruby2.3-tcltk i386 2.3.3-1+deb9u2 [426 kB]              
Get:8 http://ftp.us.debian.org/debian stretch/main i386 ruby-tcltk all 1:2.3.3 [9,334 B]                        
Get:9 http://ftp.us.debian.org/debian stretch/main i386 ruby-full all 1:2.3.3 [9,420 B]                         
Fetched 5,915 kB in 32s (183 kB/s)                                                                              
debconf: unable to initialize frontend: Dialog
debconf: (Dialog frontend requires a screen at least 13 lines tall and 31 columns wide.)
debconf: falling back to frontend: Readline
Selecting previously unselected package libgmpxx4ldbl:i386.
(Reading database ... 425666 files and directories currently installed.)
Preparing to unpack .../0-libgmpxx4ldbl_2%3a6.1.2+dfsg-1_i386.deb ...
Unpacking libgmpxx4ldbl:i386 (2:6.1.2+dfsg-1) ...
Selecting previously unselected package libgmp-dev:i386.
Preparing to unpack .../1-libgmp-dev_2%3a6.1.2+dfsg-1_i386.deb ...
Unpacking libgmp-dev:i386 (2:6.1.2+dfsg-1) ...
Selecting previously unselected package ruby2.3-doc.
Preparing to unpack .../2-ruby2.3-doc_2.3.3-1+deb9u2_all.deb ...
Unpacking ruby2.3-doc (2.3.3-1+deb9u2) ...
Selecting previously unselected package ri.
Preparing to unpack .../3-ri_1%3a2.3.3_all.deb ...
Unpacking ri (1:2.3.3) ...
Selecting previously unselected package ruby2.3-dev:i386.
Preparing to unpack .../4-ruby2.3-dev_2.3.3-1+deb9u2_i386.deb ...
Unpacking ruby2.3-dev:i386 (2.3.3-1+deb9u2) ...
Selecting previously unselected package ruby-dev:i386.
Preparing to unpack .../5-ruby-dev_1%3a2.3.3_i386.deb ...
Unpacking ruby-dev:i386 (1:2.3.3) ...
Selecting previously unselected package ruby2.3-tcltk.
Preparing to unpack .../6-ruby2.3-tcltk_2.3.3-1+deb9u2_i386.deb ...
Unpacking ruby2.3-tcltk (2.3.3-1+deb9u2) ...
Selecting previously unselected package ruby-tcltk.
Preparing to unpack .../7-ruby-tcltk_1%3a2.3.3_all.deb ...
Unpacking ruby-tcltk (1:2.3.3) ...
Selecting previously unselected package ruby-full.
Preparing to unpack .../8-ruby-full_1%3a2.3.3_all.deb ...
Unpacking ruby-full (1:2.3.3) ...
Setting up ruby2.3-doc (2.3.3-1+deb9u2) ...
Setting up ruby2.3-tcltk (2.3.3-1+deb9u2) ...
Setting up ri (1:2.3.3) ...
Setting up ruby-tcltk (1:2.3.3) ...
Processing triggers for libc-bin (2.24-11+deb9u3) ...
ldconfig: file /usr/lib/i386-linux-gnu/libQt5WebKit.so.5 is truncated

ldconfig: file /usr/lib/i386-linux-gnu/libQt5WebKit.so.5.7 is truncated

ldconfig: file /usr/lib/i386-linux-gnu/libQt5WebKit.so.5.7.1 is truncated

Setting up libgmpxx4ldbl:i386 (2:6.1.2+dfsg-1) ...
Setting up libgmp-dev:i386 (2:6.1.2+dfsg-1) ...
Setting up ruby2.3-dev:i386 (2.3.3-1+deb9u2) ...
Setting up ruby-dev:i386 (1:2.3.3) ...
Setting up ruby-full (1:2.3.3) ...
Processing triggers for libc-bin (2.24-11+deb9u3) ...
ldconfig: file /usr/lib/i386-linux-gnu/libQt5WebKit.so.5 is truncated

ldconfig: file /usr/lib/i386-linux-gnu/libQt5WebKit.so.5.7 is truncated

ldconfig: file /usr/lib/i386-linux-gnu/libQt5WebKit.so.5.7.1 is truncated

Fetching: equatable-0.5.0.gem (100%)
Successfully installed equatable-0.5.0
Fetching: necromancer-0.4.0.gem (100%)
Successfully installed necromancer-0.4.0
Fetching: tty-color-0.4.2.gem (100%)
Successfully installed tty-color-0.4.2
Fetching: pastel-0.7.1.gem (100%)
Successfully installed pastel-0.7.1
Fetching: tty-screen-0.5.1.gem (100%)
Successfully installed tty-screen-0.5.1
Fetching: unicode_utils-1.4.0.gem (100%)
Successfully installed unicode_utils-1.4.0
Fetching: unicode-display_width-1.1.3.gem (100%)
Successfully installed unicode-display_width-1.1.3
Fetching: verse-0.5.0.gem (100%)
Successfully installed verse-0.5.0
Fetching: tty-table-0.8.0.gem (100%)
Successfully installed tty-table-0.8.0
Fetching: tty-cursor-0.4.0.gem (100%)
Successfully installed tty-cursor-0.4.0
Fetching: wisper-1.6.1.gem (100%)
Successfully installed wisper-1.6.1
Fetching: tty-prompt-0.12.0.gem (100%)
Successfully installed tty-prompt-0.12.0
Fetching: httpclient-2.8.3.gem (100%)
Successfully installed httpclient-2.8.3
Fetching: neocities-0.0.9.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing neocities:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.3.0/gems/neocities-0.0.9/ext
/usr/bin/ruby2.3 mkrf_conf.rb

current directory: /var/lib/gems/2.3.0/gems/neocities-0.0.9/ext
/usr/bin/ruby2.3 -rubygems /usr/share/rubygems-integration/all/gems/rake-10.5.0/bin/rake RUBYARCHDIR=/var/lib/gems/2.3.0/extensions/x86-linux/2.3.0/neocities-0.0.9 RUBYLIBDIR=/var/lib/gems/2.3.0/extensions/x86-linux/2.3.0/neocities-0.0.9
/usr/bin/ruby2.3: No such file or directory -- /usr/share/rubygems-integration/all/gems/rake-10.5.0/bin/rake (LoadError)

rake failed, exit code 1

Gem files will remain installed in /var/lib/gems/2.3.0/gems/neocities-0.0.9 for inspection.
Results logged to /var/lib/gems/2.3.0/extensions/x86-linux/2.3.0/neocities-0.0.9/gem_make.out
```
The log file reads:
```
current directory: /var/lib/gems/2.3.0/gems/neocities-0.0.9/ext
/usr/bin/ruby2.3 mkrf_conf.rb

current directory: /var/lib/gems/2.3.0/gems/neocities-0.0.9/ext
/usr/bin/ruby2.3 -rubygems /usr/share/rubygems-integration/all/gems/rake-10.5.0/bin/rake RUBYARCHDIR=/var/lib/gems/2.3.0/extensions/x86-linux/2.3.0/neocities-0.0.9 RUBYLIBDIR=/var/lib/gems/2.3.0/extensions/x86-linux/2.3.0/neocities-0.0.9
/usr/bin/ruby2.3: No such file or directory -- /usr/share/rubygems-integration/all/gems/rake-10.5.0/bin/rake (LoadError)

rake failed, exit code 1
```

**Comment by dotcomboom** [comment]
I installed rake 12.3.1 using gem (`sudo gem install rake`) and it appears to have worked.

**Comment by dotcomboom** [comment]
https://github.com/neocities/neocities-ruby/pull/15

**Comment by kyledrake** [comment]
Will be resolved in 0.0.12
